src/backend/.config/config.local.json

```diff
@@ -0,0 +1,5 @@
+{
+  "feature_flags": {
+    "PIA_INTAKE_FORM_FLAG": true
```

Comment:
I think for dev/test/prod these flags are set in the OC console. This flag is for local dev, so think it could easily be different than dev/test/prod. Most likely though, every flag will be true in all of local/dev/test and might only ever be off in prod.